addon/utils/geometry.js

```diff
@@ -0,0 +1,17 @@
+export function getRectForNode(node) {
+  if (!(node instanceof Element)) {
+    return {
+      top: 0,
+      left: 0,
```

Comment:
These `top` and `left` values default to `0` in the React version of this, but we need to specify them explicitly here otherwise they're undefined which breaks the sticky positioning.

Reply:
TypeScript I think...

Reply:
I realise we didn't do this before, but I don't the fact that the code uses TypeScript should be a problem....since that code will be compiled to JS no?
likely adding `@shopify/javascript-utilities` as a dep and  `import { getRectForNode } from '@shopify/javascript-utilities/geometry';` should work out of the box

Reply:
Seems to be working, just pushed the changes 👍